.travis.yml

```diff
@@ -40,7 +40,7 @@ before_script:
   - mypy syft || echo "Type errors found, continuing build..."
 script:
   - coverage run -m pytest test
-  - coverage report --omit="syft/grid/*","syft/frameworks/torch/nn/rnn.py" --ignore-errors --fail-under 95 -m
+  - coverage report --omit="syft/grid/*","syft/frameworks/torch/nn/rnn.py", "versioneer.py" --ignore-errors --fail-under 95 -m
```

Comment:
I think the space before `"versioneer.py"` is causing problems with the coverage checks, which may be what's making Travis weird for this PR.

Reply:
It does not seem to be the case.